cdhweb/pages/templates/cdhpages/blocks/pull_quote.html

```diff
@@ -1,6 +1,12 @@
 {% load wagtailcore_tags %}
 
-<div>{{ self.quote|richtext }}</div>
-{% if self.attribution %}
-    <div>{{ self.attribution|richtext }} </div>
-{% endif %}
\ No newline at end of file
+<blockquote class="pull-quote">
+    <div class="pull-quote__quote rich-text">{{ self.quote|richtext }}</div>
+    {% if self.attribution %}
+        {% comment %}
+            NOTE: deliberately not using a `<cite>` here, as the CMS field is richtext, 
```

Comment:
the use of `<cite>` in a blockquote is questionable anyway as it refers to the title of a work, not an author. I'm backing this horse:
```
<figure>
  <blockquote></blockquote>
  <figcaption></figcaption>
</figure>
```
Though I wish someone over there in standards land would just make a decree once and for all.

Reply:
Oh good call! Have updated DOM.